src/mscorlib/src/System/StubHelpers.cs

```diff
@@ -46,7 +46,8 @@ unsafe static internal byte ConvertToNative(char managedChar, bool fBestFit, boo
 
         static internal char ConvertToManaged(byte nativeChar)
         {
-            byte[] bytes = new byte[1] { nativeChar };
+            Span<byte> bytes = stackalloc byte[1];
+            bytes[0] = nativeChar;
             string str = Encoding.Default.GetString(bytes);
```

Comment:
Same question here

Reply:
This is in corelib. Presumably it can just use the Span(ref T, int) ctor. Or it could use DangerousCreate that just calls it.

Reply:
I was trying to figure out why DangerousCreate wasn't showing up in intellisense:

```C#
[EditorBrowsable(EditorBrowsableState.Never)]
public static Span<T> DangerousCreate(object obj, ref T objectData, int length);
```

This totally trolled me

Reply:
@KrzysztofCwalina, @ahsonkhan, why is this marked as EditorBrowsableState.Never?  We seem to be too quick to do that.  It's already named `Dangerous`... why do we need to hide it?  Even if we wanted to hide it from the IntelliSense list, marking it as Never means that once you type out "DangerousCreate" and the open paren, you still don't get IntelliSense help with the argument list, making it difficult to use even after you've discovered it.